app/build.gradle

```diff
@@ -20,8 +20,10 @@ android {
         applicationId "nz.org.cacophony.birdmonitor"
         minSdkVersion project.ext.minimumSdkVersion
         targetSdkVersion 28
-        versionCode generateVersionCode() 
-        versionName generateVersionName()
+        versionCode 150010605
+        versionName "1.6.5"
+//        versionCode generateVersionCode()
+//        versionName generateVersionName()
```

Comment:
If the generatedVersionCode/Name won't be used anymore probably best to delete it and the functions.